tests/foreman/cli/test_puppetclass.py

```diff
@@ -37,10 +37,7 @@ def make_puppet():
 
 @pytest.mark.tier2
 @pytest.mark.upgrade
-@pytest.mark.skipif(
-    not settings.repos_hosting_url,
-    reason="repos_hosting_url is not defined in robottelo.properties",
-)
+@pytest.mark.skipif((not settings.repos_hosting_url), reason="Missing repos_hosting_url")
```

Comment:
reason="Missing repos_hosting_url" 
This one has double quotation marks, all the others single.  Does it matter?

Reply:
oops sorry, I was too late